include/alpaka/core/Common.hpp

```diff
@@ -97,7 +97,7 @@
 //! In contrast to ordinary variables, you can not define such variables
 //! as static compilation unit local variables with internal linkage
 //! because this is forbidden by CUDA.
-#if(BOOST_LANG_CUDA && BOOST_ARCH_PTX) || (BOOST_LANG_HIP && (BOOST_ARCH_HSA || BOOST_ARCH_PTX))
```

Comment:
`BOOST_ARCH_PTX` is derived from `CUDA_ARCH` which is not defined on the host side, therefore the guard on host side was always false for CUDA, therefore the qualifier of `ALPAKA_STATIC_ACC_MEM_CONSTANT` was wrong.
For unknown reason, CUDA accepted it but this is against any CUDA example code and IMO we had luck that it worked before.